# Custom message for navigation goal with speech
geometry_msgs/Vector3 nav2_goal
string nav2_movement
string speech_text
